addons/event_booth_sale/models/product.py

```diff
@@ -7,25 +7,20 @@
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
-    detailed_type = fields.Selection(selection_add=[
+    service_tracking = fields.Selection(selection_add=[
         ('event_booth', 'Event Booth'),
-    ], ondelete={'event_booth': 'set service'})
+    ], ondelete={'event_booth': 'set no'})
```

Comment:
```suggestion
    ], ondelete={'event_booth': 'set default'})
```
There is a generic word to specify the fallback on default value ;).